app/src/main/res/values/strings.xml

```diff
@@ -146,7 +146,7 @@
     <string name="syncing">Syncing...</string>
     <string name="syncing_complete">Syncing complete</string>
     <string name="two_step_login">Two-step login</string>
-    <string name="unlock_with">Unlock with %1$s</string>
+    <string name="unlock_with_biometrics">Unlock with biometrics</string>
```

Comment:
You caught us in the middle of consolidating all strings into the `ui` module. Can you update this string in `ui/src/main/res/values/strings.xml` as well?